pkg/cloud-provider/cloudapi/azure/azure_account_mgmt.go

```diff
@@ -103,25 +104,13 @@ func extractSecret(c client.Client, s *v1alpha1.SecretReference) (*v1alpha1.Azur
 	return cred, nil
 }
 
-// getVnetAccount returns first found account config to which this vnet id belongs.
-func (c *azureCloud) getVnetAccount(vpcID string) internal.CloudAccountInterface {
-	accCfgs := c.cloudCommon.GetCloudAccounts()
-	if len(accCfgs) == 0 {
-		return nil
+func extractNamespacedName(accountID string) (types.NamespacedName, error) {
```

Comment:
Find a common pkg to place this function. the implementation is same for both AWS and Azure, so we should just have 1 instance of it